archetypes/default.md

```diff
@@ -1,6 +1,5 @@
----
-title: "{{ replace .Name "-" " " | title }}"
-date: {{ .Date }}
-draft: true
----
-
++++
+date = '{{ .Date }}'
+draft = true
+title = '{{ replace .File.ContentBaseName "-" " " | title }}'
```

Comment:
Is this change required to adopt the new theme?

I would prefer to constrain changes to only the minimum required.